.changeset/strong-flowers-ring.md

```diff
@@ -0,0 +1,7 @@
+---
+'@digdir/designsystemet': minor
+---
+
+feat(cli): add --verbose option to "tokens build" for easier debugging
+fix(cli): update tokens template to reflect removal of `ingress` and renaming of `paragraph` to `body`
+fix(cli): `tokens build` crashed when run on result of `tokens create`
```

Comment:
```suggestion
Fix CLI crashing when creating tokens:
- add --verbose option to "tokens build" for easier debugging
- update tokens template to reflect removal of `ingress` and renaming of `paragraph` to `body`
- `tokens build` crashed when run on result of `tokens create`
```

Reply:
Nice, I was uncertain of the structure here. I rewrote it somewhat, see newest commit

Reply:
Looks good, I'm unsure if changeset likes two lists - but lets merge this in and see!
We can always change it later if it comes out weird